azure/scope/machine_test.go

```diff
@@ -1437,9 +1458,10 @@ func TestMachineScope_GetVMImage(t *testing.T) {
 						Name: "machine-name",
 					},
 				},
+				ClusterScoper: clusterMock,
 			},
 			want: func() *infrav1.Image {
-				image, _ := azure.GetDefaultUbuntuImage("1.20.1")
+				image, _ := svc.GetDefaultUbuntuImage(context.TODO(), "", "1.20.1")
 				return image
 			}(),
 			wantErr: false,
```

Comment:
Can we change `wantErr` to `expectedErr` as a string and then compare that with the error we're getting back from the service?